packages/story-editor/src/components/helpCenter/constants.js

```diff
@@ -150,24 +133,7 @@ export const Z_INDEX = {
   QUICK_TIP: 2,
 };
 
-export const BASE_NAVIGATION_FLOW = Object.keys(TIPS);
-
 export const POPUP_ID = 'help_center_companion';
 export const FOCUSABLE_POPUP_CHILDREN_SELECTOR = FOCUSABLE_SELECTORS.map(
   (selector) => `#${POPUP_ID} ${selector}`
 ).join(', ');
-
-export const ReadTipsType = PropTypes.shape({
-  ...Object.keys(TIPS).reduce(
-    (accum, tipKey) => ({
-      ...accum,
-      [tipKey]: PropTypes.bool,
-    }),
-    {}
-  ),
-});
-
-export const TIP_KEYS_MAP = Object.keys(TIPS).reduce((keyMap, key) => {
-  keyMap[key] = true;
-  return keyMap;
-}, {});
```

Comment:
After reading the current code on `main` I really don't understand why `TIPS_KEY_MAP` is needed.

Everywhere it's just really used with `Object.keys(TIP_KEYS_MAP)`. This could just be replaced with `Object.keys(TIPS)` or `TIPS` (or well, `TIPS + additionalTips`) directly.

Reply:
yea I think the original intention was to keep a consistent type with the read tips of `{ [k in keyof tips]: boolean }`, but I think you're right and you could substitute out `TIPS + additionalTips` it would work the same